addon/mixins/touch-action.js

```diff
@@ -37,15 +36,15 @@ export default Mixin.create({
   ignoreTouchAction: false,
 
   init() {
-    this._super();
+    this._super(...arguments);
```

Comment:
is this really necessary? init is not supposed to have arguments and this adds an extra loop to the babel output that's otherwise avoidable.